### PR TITLE
pad results with whitespace - re #190

### PR DESCRIPTION
**Pull request by indirectlylit**
Re: #190 – Padded the output to get line numbers accurate. Not sure if this is the correct strategy or not. Also did not test the effects on source maps.

### CODE REVIEW

**Comment by indirectlylit** [comment]
Unfortunately, it looks like this change does not work because it doesn't take into account multiple `style` blocks. I'll look into that.

**Comment by indirectlylit** [comment]
Actually that's not true: it does report proper line numbers, even with multiple style blocks. That said, the whole strategy feels a bit hacky, as though it should be handled with source maps or similar. Unfortunately that's outside my expertise.

Any feedback would be appreciated.

File: test/test.js [hunk 1]
```diff
@@ -215,7 +215,8 @@ describe('vue-loader', function () {
     }), function (err) {
       expect(err).to.be.null
       getFile('test.output.css', function (data) {
-        expect(data).to.contain('h1 {\n  color: #f00;\n}\n\nh2 {\n  color: green;\n}')
+        console.log(data);
```
Comment (yyx990803) [inline test/test.js]: can we remove the log here?

File: lib/parser.js [hunk 2]
```diff
@@ -128,7 +131,7 @@ module.exports = function (content, filename, needMap) {
           map.addMapping({
             source: filenameWithHash,
             original: {
-              line: index + 1 + lineOffset,
+              line: index + 1,
```
Comment (yyx990803) [inline lib/parser.js]: it seems after this change, `lineOffset` is only needed for the padding calculation, so it should be declared directly in the `else` block above.

**Comment by indirectlylit** [comment]
Thanks for the comments - fixed.

This PR has improved parse and lint errors for us, but I'm not convinced it's the most correct solution to the problem. For example: when source maps are enabled, chrome dev tools shows the padded, style-only file rather than the original `.vue` file.

**Comment by yyx990803** [comment]
Yeah, agreed. I'll merge this for now as a temporary workaround, and will see if we can get better sourcemap handling in the future. Thanks!

**Pull request merged**